### PR TITLE
Add APT deps guide & extra package explanations

### PR DESCRIPTION
**Pull request by furiosamg**
[Updated document](https://furiosa-ai.github.io/furiosa-models/PR-183/getting_started/)
APT 의존성 설치와 python extra package 에 대한 설명을 추가했습니다.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -37,6 +51,23 @@ pip install 'furiosa-models'
     pip install ./furiosa-models
     ```
 
+### Extra packages
+
+To enable the use of the `furiosa models serve` command, you need to install the `furiosa-serving` Python package.
+You can achieve this by specifying the serving extra package when installing `furiosa-models`.
+
+```shell
+pip install 'furiosa-models[serving]`
```
Comment (libc-furiosa) [inline docs/getting_started.md:60]: Quote character mismatched.

```suggestion
pip install 'furiosa-models[serving]'
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -37,6 +51,23 @@ pip install 'furiosa-models'
...
+```
+
+If you require additional packages related to compilation, you can install the full extra package, which includes all relevant dependencies.
+
+```shell
+pip install 'furiosa-models[full]`
```
Comment (libc-furiosa) [inline docs/getting_started.md:66]:
```suggestion
pip install 'furiosa-models[full]'
```

**Review by libc-furiosa — COMMENTED** [review]
Please check comments.

File: docs/getting_started.md (hunk 1 above)
Comment (furiosamg) [reply docs/getting_started.md]: Thanks! Applied in bb4a494

File: docs/getting_started.md (hunk 2 above)
Comment (furiosamg) [reply docs/getting_started.md]: Thanks! Applied in bb4a494

**Review by libc-furiosa — APPROVED** [review]
Thanks. LGTM!

**Pull request merged**